inspirehep/config.py

```diff
@@ -1188,3 +1188,9 @@ def _(x):
     "stat.OT": "Other",
     "stat.TH": "Other"
 }
+
+# Import local config file if it is present.
+try:
+    from inspirehep.config_local import *
```

Comment:
Ok. I see it is added above `https://github.com/inspirehep/inspire-next/pull/1251/files/ae94900a7ca8c08f1ddb50f57244c5415da95919#diff-a084b794bc0759e7a6b77810e01874f2R6` in `.gitignore` I personally don't see the need for it and the least lines the better but if other people find it useful its ok.